content/en/users/cloud-compute/openstack/_index.md

```diff
@@ -29,14 +29,39 @@ of this guide will focus on CLI/API access.
 
 ## Installation
 
-TBC
+The OpenStack client is a command-line client for OpenStack that brings the
+command set for Compute, Identity, Image, Object Storage and Block Storage
+APIs together in a single shell with a uniform command structure.
+
+> If installing on Windows, as there are non-pure Python packages needed for
+> installation, please make sure [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/)
+> are installed, with the following options selected:
+>
+> - C++ CMake tools for Windows
+> - C++ ATL for latest v142 build tools (x86 & x64)
+> - Testing tools core features - Build Tools
+> - Windows 10 SDK (`<latest`>)
+>
+> In case you prefer to use non-Microsoft alternatives for building non-pure
+> packages, please see [here](https://wiki.python.org/moin/WindowsCompilers).
+
+Installation of the OpenStack client can be done using:
 
 ```shell
+pip install --upgrade setuptools
```

Comment:
is this windows specific?

Reply:
yes, I put Windows specific steps in blockquotes

Reply:
I would remove the steps about setuptools upgrade, it doesn't seems like something we want to cover in this docs, I would stick the minimal and even more as here those are OS-agnostic steps.

Reply:
done